docs/canonicalk8s/charm/howto/install/install-terraform.md

```diff
@@ -113,12 +73,19 @@ terraform {
   required_providers {
     juju = {
       source  = "juju/juju"
-      version = "~> 0.14.0"
+      version = ">= 0.14.0, < 1.0.0"
     }
   }
 }
 ```
 
+### Cloud Integrations
+
+The bundle will support various cloud integrations, to integrate {{ product }}
```

Comment:
I am a bit confused by this paragraph. Is the gist you can integrate Canonical K8s with different cloud substrates. It doesn't integrate by default though and you must specify which cloud you want to integrate with?

Reply:
Yes, that's accurate.  let me see if i can clarify that.

Reply:
@nhennigan does this resolve the questions

Reply:
Yes thank you :)